tests/test_ble_interface_core.py

```
@@ -411,9 +417,248 @@ def test_ble_interface_defaults_auto_reconnect_disabled(
+        iface,
+        "findDevice",
+        lambda _address: _create_ble_device("AA:BB:CC:DD:EE:FF", "Meshtastic"),
+    )
+    monkeypatch.setattr("meshtastic.interfaces.ble.interface.sys.platform", "linux")
+    monkeypatch.setattr(
+        "meshtastic.interfaces.ble.interface.shutil.which",
+        lambda _name: "/usr/bin/bluetoothctl",
+    )
+    monkeypatch.setattr(
+        "meshtastic.interfaces.ble.interface.subprocess.run",
+        lambda *args, **kwargs: SimpleNamespace(
```

Ruff (0.15.2)

[warning] Unused lambda argument: `args`

(ARG005)